res2df/pvt.py

```diff
@@ -239,14 +239,14 @@ def df(
 
 
 def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
-    """Set up sys.argv parsers for parsing Eclipse deck or
-    include files into dataframes
+    """Set up sys.argv parsers for parsing :term:`.DATA file` or
+    :term:`include files <include file>` into dataframes
 
     Arguments:
         parser (ArgumentParser or subparser): parser to fill with arguments
     """
     parser.add_argument(
-        "DATAFILE", help="Name of Eclipse DATA file or file with PVT keywords."
+        "DATAFILE", help="Name of reservoir .DATA file or file with PVT keywords."
```

Comment:
parser.add_argument(
"DATAFILE", help="Name of the .DATA input file for the reservoir simulator or file with PVT keywords."